app/src/main/java/org/schabi/newpipe/player/ServicePlayerActivity.java

```diff
@@ -164,10 +168,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 return true;
             case R.id.action_switch_main:
                 return switchTo(MainVideoPlayer.class);
+            case R.id.action_set_timer:
+                try {
+                    updateTimer(this);
+                } catch (Exception e) {
+                    e.printStackTrace();
+                    Toast.makeText(getApplicationContext(), getString(R.string.error_setting_timer), Toast.LENGTH_SHORT).show();
+                }
```

Comment:
Not being able to set the timer due to an unknown exception is a critical error, so it is ok for it to crash the app and open the bug report screen. So this whole piece should be converted into:
`updateTimer(this);`

Reply:
Done